static/templates/reports/pac.hbs

```diff
@@ -5,9 +5,16 @@
       (This report has been amended)
```

Comment:
Let's remove this part from all these templates. Redundant with the other new info.